server/controllers/user.js

```diff
@@ -21,3 +21,16 @@ exports.searchUsers = asyncHandler(async (req, res, next) => {
 
   res.status(200).json({ users: users });
 });
+
+// @route GET /users/find
+// @desc Find your user info
+exports.getUserInfo = asyncHandler(async (req, res, next) => {
+  const id = req.query._id;
```

Comment:
It would be good to validate request query and return a bad request status if the required queries are not supplied.

Reply:
this is an automatic api call when you pull up your profile page

Reply:
I implemented validation on the two search calls